samples/drivers/led_is31fl3216a/boards/lpcxpresso55s28.overlay

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
Suggestion: consider providing an overlay for the standard arduino I2C connector.

See samples/sensor/bme280/arduino_i2c.overlay for an example for a sensor device.

This makes it easy to test on a wide variety of vendor boards that have this connector in their BOARD.dts out of the box.

Reply:
Unfortunately I haven't had time to improve this PR, I hope I will be able to do it by the end of the next week.
Thanks for your feedbacks and patience!